<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VerticalMapTargetRegionWidget</class>
 <widget class="QWidget" name="VerticalMapTargetRegionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>117</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignTop">
    <widget class="QGroupBox" name="targetGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>706</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Target Region</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="3">
       <widget class="UtQtLengthValueEdit" name="mAltitudeMaxEdit">
        <property name="maximumSize">
         <size>
          <width>134</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Altitude</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="UtQtLengthValueEdit" name="mAltitudeMinEdit">
        <property name="maximumSize">
         <size>
          <width>134</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="UtQtLengthValueEdit" name="mGroundRangeMaxEdit">
        <property name="maximumSize">
         <size>
          <width>134</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="UtQtLengthValueEdit" name="mAltitudeStepEdit">
        <property name="maximumSize">
         <size>
          <width>134</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="UtQtLengthValueEdit" name="mGroundRangeMinEdit">
        <property name="maximumSize">
         <size>
          <width>134</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Ground Range</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="UtQtLengthValueEdit" name="mGroundRangeStepEdit">
        <property name="maximumSize">
         <size>
          <width>134</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Maximum</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Step</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Minimum</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UtQtLengthValueEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtUnitValueEdit.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mGroundRangeMinEdit</tabstop>
  <tabstop>mGroundRangeMaxEdit</tabstop>
  <tabstop>mGroundRangeStepEdit</tabstop>
  <tabstop>mAltitudeMinEdit</tabstop>
  <tabstop>mAltitudeMaxEdit</tabstop>
  <tabstop>mAltitudeStepEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
